spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/iam/PublicKeyResolver.java

```diff
@@ -0,0 +1,18 @@
+package org.eclipse.dataspaceconnector.spi.iam;
```

Comment:
There's another `PublicKeyResolver` in the `oauth2-spi` module with the same interface.